manifests/params.pp

```diff
@@ -121,8 +121,8 @@
 
       # Archlinux doesn't have a client-package but has a libs package which
       # pulls in postgresql server
-      $client_package_name    = pick($client_package_name, 'postgresql')
-      $server_package_name    = pick($server_package_name, 'postgresql-libs')
+      $client_package_name    = pick($client_package_name, 'postgresql-libs')
```

Comment:
this change is correct. what you need in mind: postgresql pulls in postgresql-libs, I don't know if this will now cause some conflicts.

Reply:
That's well understood - and I've never seen a database server running postgresql or mysql that doesn't have the psql/mysql command line tool installed.

Reply:
@tobixen can you please rebase?